### PR TITLE
Update en correct .dnn manifest

### PR DESCRIPTION
**Pull request by EPTamminga**
lose #167

### CODE REVIEW

File: Resources/ReleaseNotes.txt [hunk 1]
```diff
@@ -15,12 +15,18 @@ Belangrijkste verbeteringen in deze release van het nl-NL taalpakket:
 Aanvullen van de vertaling van Prompt
 </li>
 <li style="margin-left:20px;">
+Ontbrekende installatie regels voor enkele nieuwe nl-NL.esx bestanden aan .dnn manifest toegevoegd
```
Comment (aspelt) [inline Resources/ReleaseNotes.txt:18]: Ontbrekende installatie regels voor enkele nieuwe nl-NL.resx bestanden aan .dnn manifest toegevoegd

**Review by aspelt — APPROVED** [review]
Looks good to me!

File: Resources/DesktopModules/Admin/Dnn.PersonaBar/Modules/Dnn.Prompt/App_LocalResources/prompt.nl-NL.resx [hunk 2]
```diff
@@ -12,1095 +12,10 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="CommandNotFound.Text" xml:space="preserve">
```
Comment (valadas) [inline Resources/DesktopModules/Admin/Dnn.PersonaBar/Modules/Dnn.Prompt/App_LocalResources/prompt.nl-NL.resx:15]: Tere are 1089 removed line in this file, is this on purpose ?

**Review by valadas — CHANGES_REQUESTED** [review]

File: Resources/DesktopModules/Admin/Dnn.PersonaBar/Modules/Dnn.Prompt/App_LocalResources/prompt.nl-NL.resx (hunk 2 above)
Comment (EPTamminga) [reply Resources/DesktopModules/Admin/Dnn.PersonaBar/Modules/Dnn.Prompt/App_LocalResources/prompt.nl-NL.resx:15]: Yes,  we used the translator tool from Peter Donker for the prompt cleaning and reviewed that manually comparing it with the en-US version of prompt.
All promote text elements are move to the PB modules and are no longer in the prompt .resx file itself.

**Review by aspelt — APPROVED** [review]

File: Resources/DesktopModules/Admin/Dnn.PersonaBar/Modules/Dnn.Prompt/App_LocalResources/prompt.nl-NL.resx (hunk 2 above)
Comment (EPTamminga) [reply Resources/DesktopModules/Admin/Dnn.PersonaBar/Modules/Dnn.Prompt/App_LocalResources/prompt.nl-NL.resx:15]: @aspelt Can you confirm the removal of a lot of text elements in prompt resx?

**Review by valadas — APPROVED** [review]
great

**Pull request merged**